base/src/dispatcher/dispatcher.ts

```diff
@@ -129,20 +130,20 @@ export class Dispatcher implements ZLUX.Dispatcher {
 
    runHeartbeat():void {
      let dispatcherHeartbeatFunction = () => {
-     this.log.debug('Recognizers: ', this.recognizers);
-     this.log.debug("dispatcher heart beat");
-     this.log.debug("instances for Types = ", this.instancesForTypes);
-     this.log.debug("indexed recognizers = ", this.indexedRecognizers);
+     this.log.debug("ZWED5020I", this.recognizers); //this.log.debug('Recognizers: ', this.recognizers);
```

Comment:
It will be very difficult to understand what is being logged with just a message ID. As a first step I want to keep the messages there so we can experience doing support with the calls. We can delete them later if the cost outweighs the benefit.